README.md

```diff
@@ -59,10 +59,12 @@ var css = fs.readFileSync("input.css", "utf8")
 
 ### Configuring rules
 
-[Rules](docs/rules.md) are configured within the `rules` key of the config. Each rule can be turned off or on:
+[Rules](docs/rules.md) are configured within the `rules` key of the config.
+Like [ESLint](http://eslint.org/docs/user-guide/configuring#configuring-rules), each rule can be turned off or on:
 
 * 0 - turn the rule off
-* 2 - turn the rule on
+* ~~1 - turn the rule on as a warning (doesn't affect exit code)~~ @todo
+* 2 - turn the rule on as an error (~~exit code is 1 when triggered~~ might affect exit code to 1 in a near future)
```

Comment:
I think people can read really quickly and won't see the note (like me, I skipped the note under `# Installation`...).
We can update with the link the issue with a similar message, but I think it is better to keep the list with 0,1,2 in this order.